library/docs/useDirections.md

```diff
@@ -33,11 +37,12 @@ interface DirectionsProps {
 
 ## Return value
 
-Returns a [`directionsService`](https://developers.google.com/maps/documentation/javascript/reference/directions#DirectionsService) instance to use directly, `findRoute` which returns a route and `findAndRenderRoute` which also renders the route on the map. When using `findAndRenderRoute`, the `renderOnMap` prop should be set to `true`. `renderRouteOfIndex` can be used to render a specific route of `google.maps.DirectionsResult` returned by `findRoute` or `findAndRenderRoute`.
+Returns a [`directionsService`](https://developers.google.com/maps/documentation/javascript/reference/directions#DirectionsService) instance and a [`directionsRenderer`](https://developers.google.com/maps/documentation/javascript/reference/directions#DirectionsRenderer) instance to use directly, `findRoute` which returns a route and `findAndRenderRoute` which also renders the route on the map. When using `findAndRenderRoute`, the `renderOnMap` prop should be set to `true`. `renderRouteOfIndex` can be used to render a specific route of `google.maps.DirectionsResult` returned by `findRoute` or `findAndRenderRoute`.
```

Comment:
Maybe list them below each other with either bullet points or numbers and the descriptions for each. Could be easier to read